diff -urbB libgnomedb-1.9.100.org/libgnomedb/Makefile.am libgnomedb-1.9.100/libgnomedb/Makefile.am
--- libgnomedb-1.9.100.org/libgnomedb/Makefile.am	2005-09-26 12:37:39.000000000 +0200
+++ libgnomedb-1.9.100/libgnomedb/Makefile.am	2005-10-24 23:05:46.088559250 +0200
@@ -112,9 +112,6 @@
 	sel-queries.h \
 	sel-tables.h 
 
-gnomedbincludedir=$(includedir)/libgnomedb-$(GNOMEDB_MAJOR_VERSION).$(GNOMEDB_MINOR_VERSION)/libgnomedb
-gnomedbinclude_HEADERS=$(gnomedb_headers)
-
 if BUILD_WITH_GNOME
 withgnome_sources = \
 	gnome-db-data-source-selector.c \
@@ -123,10 +120,15 @@
 	gnome-db-login.c \
 	gnome-db-login-dialog.c \
 	gnome-db-util.h 
+gnome_gnomedb_headers = gnome-db-util.h
 else
 withgnome_sources = 
+gnome_gnomedb_headers =
 endif
 
+gnomedbincludedir=$(includedir)/libgnomedb-$(GNOMEDB_MAJOR_VERSION).$(GNOMEDB_MINOR_VERSION)/libgnomedb
+gnomedbinclude_HEADERS=$(gnomedb_headers) $(gnome_gnomedb_headers)
+
 libgnomedb_3_la_SOURCES = \
 	$(gnomedb_headers) \
 	gnome-db-combo.c \
